packages/lib/gpcircuits/test/common.ts

```diff
@@ -16,6 +16,9 @@ export const circomkit = new Circomkit({
 export const privateKey =
   "0001020304050607080900010203040506070809000102030405060708090001";
 
+export const privateKey2 =
```

Comment:
This is going to need an update when it rebases onto my PR switching keys to Base64.